app/services/analytics.rb

```diff
@@ -1,3 +1,5 @@
+# frozen_string_literal: true
```

Comment:
No, I didn't change ALL THE NAMES, I just got rid of the `.freeze` at the end of each line in favor of this magic comment.

The only new events are IDV_DOC_AUTH_EXCEPTION_VISITED and IDV_DOC_AUTH_WARNING_VISITED